server/usecases/board/github.go

```diff
@@ -2,18 +2,26 @@ package board
 
 import (
 	"database/sql"
+	"encoding/json"
 	"strings"
+	"time"
 
+	"github.com/go-redis/redis"
 	"github.com/google/go-github/github"
 	"github.com/h3poteto/fascia/config"
 	"github.com/h3poteto/fascia/lib/modules/hub"
 	"github.com/h3poteto/fascia/lib/modules/logging"
+	rediscli "github.com/h3poteto/fascia/lib/modules/redis"
 	"github.com/h3poteto/fascia/server/domains/list"
 	"github.com/h3poteto/fascia/server/domains/project"
 	"github.com/h3poteto/fascia/server/domains/task"
 	"github.com/pkg/errors"
 )
 
+type Redis struct {
```

Comment:
__[Golint]__ exported type Redis should have comment or be unexported ([link](https://sider.review/gh/repos/50192141/pulls/456#issue-3827454))

<sub>posted by Sider</sub>